src/core/watchSources.ts

```diff
@@ -12,40 +12,50 @@ const eventMapping: Record<string, (params: WatchEventParams) => void> = {
 };
 
 export default function watchSources({ config, sourcesMap }: WatchSourcesParams): void {
-  const sourcePaths = config.copy.from
-    .filter(source => source.watch)
-    .map(source => getFilesGlob(source.path));
+  const sourcePaths = config.copy.from.filter(source => source.watch);
```

Comment:
maybe its good to add optional chaining

Reply:
`copy.from` is a required part of the configuration, and without it, other things would break before this line of code. I think it's safe to leave it as it is.